src/components/RoutesOverview.tsx

```diff
@@ -36,12 +42,44 @@ export default function RoutesOverview({
   onRouteClick: (routeIndex: number, evt: React.MouseEvent) => void;
 }) {
   const intl = useIntl();
+  const dispatch = useDispatch();
   const SPACE = ' ';
+  const os = useMemo(() => Bowser.parse(navigator.userAgent).os.name, []);
+
+  const shareText = intl.formatMessage({
+    defaultMessage: 'Share',
+    description:
+      'Button. When clicked, lets you share a set of directions, ' +
+      'such as by copying a URL to the clipboard.',
+  });
 
   let containsTransitLeg = false;
 
   const outOfAreaMsg = _outOfAreaMsg(intl, outOfAreaStart, outOfAreaEnd);
 
+  const handleShareClick = async (evt: React.MouseEvent) => {
+    try {
+      await navigator.clipboard.writeText(String(document.location));
```

Comment:
Did you consider using the [share method](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share) to pop open the OS specific share menu? Then doing something like this that calls back on the copy/paste technique if your on desktop or whatever device doesn't support the method?
```suggestion
      if (navigator.share) {
        await navigator.share({
          title: "BikeHopper",
          text: "Route through SF Bay Area",
          url: String(document.location),
        });
      } else {
        await navigator.clipboard.writeText(String(document.location));
      }
```